packages/notebook-viewer/requirements.txt

```diff
@@ -2,3 +2,4 @@ Flask==2.1.2
 Jinja2==3.1.2
 nbconvert==7.2.7
 nbformat==5.7.1
+Werkzeug==2.3.7
```

Comment:
I had to build the project from scratch and apparently there's an issue with how Flask specifies it's Werkzeug dependency. This [SO answer](https://stackoverflow.com/a/77214086) guided me to this solution, which caused the app to build correctly.